integration/tests/crud/creategetdelete_test.go

```diff
@@ -420,6 +420,11 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 							Name:         "test-containerd",
 							AMIFamily:    api.NodeImageFamilyAmazonLinux2,
 							InstanceType: "p2.xlarge",
+							AdditionalVolumes: []*api.VolumeMapping{
+								{
+									VolumeName: aws.String("/dev/sdb"),
+								},
+							},
```

Comment:
:eyes: are you adding the check to see it results in the volume being added? *runs*

Reply:
Nope, because it's expensive. It would fail if it didn't succeed. I would have get the asg, find out which instance is added, do a discribe instance, do a describe volumes and search for the mount. For every nodegroup and every node it would add around 3 more API calls.

Reply:
>  because it's expensive.

is 3 api calls expensive? Sorry I know I'm being really annoying, but a lot of our integration tests never check that the desired outcome actuall happens, just that we exit 0

Reply:
Considering we have a ticket to reduce the number of overwhelming API calls and the number of times it fails because of throttling issue, yes, I think 3 is a lot. :) And that's per nodegroup and per instance. If there are two instances there that will be 6. :)

Reply:
> Considering we have a ticket to reduce the number of overwhelming API calls and the number of times it fails because of throttling issue, yes, I think 3 is a lot. :)

thats for eksctl itself, not for the tests. We don't care how many API calls the tests make

Reply:
fine